src/support/modify.c

```diff
@@ -445,10 +444,12 @@ __wt_modify_apply_api(WT_CURSOR *cursor, WT_MODIFY *entries, int nentries)
 /*
  * __wt_modify_reconstruct_from_upd_list --
  *     Takes an in-memory modify and populates an update value with the reconstructed full value.
+ *
+ * The reconciliation parameter should be set to true when called from reconciliation.
```

Comment:
Comment needs fixing for the new enum.

Reply:
[cfd6173](https://github.com/wiredtiger/wiredtiger/pull/10311/commits/cfd6173e791bd0fb94c3b4c7af5a952e48394496)